README.md

```diff
@@ -127,6 +127,49 @@ pprint(annotation["page"][3])  # the data of the 4th page of "ARMS"
 #            '@xmin': 1155,
 #            '@ymax': 686,
 #            '@ymin': 595} ... ]}
+
+# (4) Preserve ordering in the annotation data
+annotation_ordered = p.get_annotation(book="ARMS", separate_by_tag=False)
+
+# In the raw XML in the Manga109 dataset, the bounding box data in the
+# `page` tag is not sorted by its annotation type, and each bounding
+# box type appears in an arbitrary order. The output dictionary of this
+# function preserves the ordering of each bounding box tag in the raw
+# XML data, mainly for data editing purposes. Note that the ordering of
+# the bounding box tags does not carry any useful information about the
+# contents of the data.
+
+# Caution: Due to the aforementioned feature, the format of the output
+# dictionary differs from the `get_annotation` function.
+
+# (4-1) Tag ordering in `page`
```

Comment:
(4-1)?

Reply:
After looking again at the readme, I thought this line was redundant, so I will be deleting it altogether.